stores/json/store.go

```diff
@@ -149,14 +149,24 @@ func (store Store) treeBranchFromJSONDecoder(dec *json.Decoder) (sops.TreeBranch
 	}
 }
 
+// Encoder to disable escaping html symbols
+// See: https://github.com/getsops/sops/issues/881
+func jsonMarshal(v interface{}) ([]byte, error) {
+    buffer := &bytes.Buffer{}
+    encoder := json.NewEncoder(buffer)
+    encoder.SetEscapeHTML(false)
+    err := encoder.Encode(v)
+    return buffer.Bytes(), err
+}
```

Comment:
Please run gofmt to reformat this file (you use spaces instead of tabs for indentation). Also you need to sign off the commit (for DCO). Besides that, I think the PR looks good.